src/lib/middleware/proxy_to_reflection.js

```diff
@@ -6,14 +6,35 @@ const httpProxy = require("http-proxy")
 const request = require("superagent")
 const proxy = httpProxy.createProxyServer()
 const { parse } = require("url")
-const { REFLECTION_URL } = process.env
+
+let ignoreList = null
+
+/**
+ * Provides a set of paths to ignore from reflection for testing purposes. This
+ * is called only on the first middleware execution to avoid unnecessary
+ * recalculations.
+ */
+function setIgnoreList() {
+  // Destructured here so it can actually be mocked in tests
+  const { REFLECTION_IGNORE } = process.env
+  ignoreList =
+    typeof REFLECTION_IGNORE === "string" ? REFLECTION_IGNORE.split(";") : []
```

Comment:
Why not `,`-delimited? I see one of those among Force's configs already. Then again, I also see a `|`-delimited config. Would prefer to have fewer varieties of configuration formats rather than more.